packages/util-utf8/src/fromUtf8.browser.ts

```diff
@@ -0,0 +1,5 @@
+import { Encoding } from "./whatwgEncodingApi";
```

Comment:
The whatwgEncodingApi is no longer needed, as TextEncoder and TextDecoder is present in typescript lib "dom"
https://github.com/microsoft/TypeScript/blob/b4024a5506253f87c61da608ed6f84ab95f7a186/lib/lib.dom.d.ts#L13930

You can remove the declaration of TextEncoder and TextDecoder too